### ISSUE TITLE
Unlink pointing between 2 accounts (transfer)

### ISSUE THREAD

**Issue by gtrhayrug** [moneymanagerex/moneymanagerex#3843]
Hi,

Any idea to unlink pointing transactions on 2 accounts ?

If I made a pointing action on a transaction in one account, automatically, pointing is made on the second account (especially for transfer or moving between account).

Therefore, when i received the bank listing operations, the total amount (debit/credit) is false on MoneyManager.

Thanks !

**Comment by tactilis** [comment]
I _think_ I understand what you are asking - but to be certain, could you describe your workflow please?

Are you entering transactions (including transfers) manually when they occur, and then, at a later date, importing the same transactions from your bank account data?

Please explain step by step _exactly_ what you are doing and give example transactions that shows what MMEX is doing wrong for you that results in the "total amount (debit/credit) being false".

**Comment by vomikan** [comment]
Related issue
https://github.com/moneymanagerex/moneymanagerex/issues/1637

**Comment by tactilis** [comment]
> Please explain step by step _exactly_ what you are doing...

@gtrhayrug posted his reply in #1637, I've copied here so it can be read in context...
 
---
Confirmed. When you are pointing transactions using bank pointing reference, especially between 2 accounts, pointing will confirmed the transaction on both accounts.

For example :

- you make a withdraw transfer between account A and account B ;
- you received bank pointing list where withdraw debit between A to B is pointed ;
- therefore when you pointed this operation (on account A), the pointing sold of account B is false because transfer on account B is automatically pointed.

So, based on proposition :
RU => pointed on account A ; non-pointed on account B ;
RR => pointed on A/B ;
UR => etc.
